### PR TITLE
Cell station

### PR DESCRIPTION
**Pull request by Deerstop**
## About the PR

Cell station is a low population (7-35) map with an aged, dilapidated theme.

Cell features:
- Tesla-focused containment field
- Comfortable brig cells and perma
- Extensive, but damaged meteor shielding, forming a partial perimeter around the station
- Command evac docking arm, with secure Centcom-only dock
- Maintenance-themed engineering department
- Shuttle workshop

## Why / Balance

Wizden lowpop gameplay suffers from a lack in map variety, which is, frankly, boring. A new station will help breathe more life into it.
The run-down aesthetic is to encourage the kind of crew cooperation in maintaining and cleaning the station that Barratry provided, but smaller in scale and completely optionally.
## Technical details

Cell station is up to date with holopads, atmos network console/sensors, and dim lights.
## Media
Current Map Render
Cell-0

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking Changes
**Changelog**
:cl:Deerstop
- add: Added Cell Station, a dingy low-pop map.

### CODE REVIEW

**Comment by Deerstop** [comment]
station isn't entirely ready yet, still need to finish renaming doors/apcs/etc, and get more feedback/a proper review

File: SpaceStation14.sln [hunk 1]
```diff
@@ -420,24 +420,24 @@ Global
 		{07CA34A1-1D37-4771-A2E3-495A1044AE0B}.DebugOpt|Any CPU.Build.0 = DebugOpt|Any CPU
 		{07CA34A1-1D37-4771-A2E3-495A1044AE0B}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{07CA34A1-1D37-4771-A2E3-495A1044AE0B}.Release|Any CPU.Build.0 = Release|Any CPU
-		{07CA34A1-1D37-4771-A2E3-495A1044AE0B}.Tools|Any CPU.ActiveCfg = Tools|Any CPU
```
Comment (TheShuEd) [inline SpaceStation14.sln:423]: remove all changes in this file

**Comment by cohanna** [comment]
this looks a little large for lowpop ngl
would be a better mid pop imo

**Comment by spanky-spanky** [comment]
> this looks a little large for lowpop ngl would be a better mid pop imo

At a glance it looks smaller than Packed.

**Comment by cohanna** [comment]
Aw I liked this one

**Comment by Deerstop** [comment]
> Aw I liked this one

it'll be back, i'm just redoing git stuff on my end to end to hopefully fix some issues

**Comment by Deerstop** [comment]
https://github.com/space-wizards/space-station-14/pull/34085 new pr for cell is open

**Pull request closed without merging**